### ISSUE TITLE
unable to run skyrim LE even basegame without mods or DLCs

### ISSUE THREAD

**Issue by Mass1milian0** [rockerbacon/modorganizer2-linux-installer#293]
so, after installing the latest release (4.2.2), the installation goes through perfectly, however whenever i launch the game i get the bethasda log then blackscreen and crash, no log in any shape or form, even with papyrus enabled, on the console, the only error i can see is:
```
ERROR: Could Not Get Primary Adapter Handle
warn; D3D9DeviceEx::SetRenderState: Unhandled render state D3DRS_CLIPPING
```
that appears both when running with and without skse

and as a final note, all the tests were done with those "plugins" enabled:
```
Skyrim.esm
Update.esm

Dawnguard.esm
HeartFires.esm
Dragonborn.esm
HighResTexturePack01.esp
HighResTexturePack02.esp
HighResTexturePack03.esp
```

disabling those plugins does not impact the terminal logs nor does it make the game launch

the proton logs don't seem to be very helpful, and are also VERY MASSIVE (17886 lines) unless request, i won't upload the log file as nobody should go through the torture of reading (or even scrolling) through 17k lines

also as i final test, i verified the installation basically unlinking mo2 from skyrim launcher on steam and launching the game after verification does launch the game, so im pretty sure that mo2 is somehow corrupting the installation of the game or preventing it from launching somehow

**Comment by rockerbacon** [comment]
Please note 4.2.2 is a pre-release and is there simply for testing. You should be using [4.2.0](https://github.com/rockerbacon/modorganizer2-linux-installer/releases/tag/4.2.0).

A quick search showed two things:
* Might be related to Vulkan or graphics card drivers: [https://steamcommunity.com/app/221410/discussions/0/3006682619671417962/?l=polish](https://steamcommunity.com/app/221410/discussions/0/3006682619671417962/?l=polish)
* Might be related to Protontricks not properly applying `d3dcompiler_43.dll` [https://github.com/ValveSoftware/Proton/issues/5165](https://github.com/ValveSoftware/Proton/issues/5165), which could have been caused by using 4.2.2. I'd recommend purging your existing prefix (as described in step 3 of [using alternative proton versions](https://github.com/rockerbacon/modorganizer2-linux-installer/blob/master/post-install.md#using-alternative-proton-versions)) and re-installing using 4.2.0. Make sure to read the installation instructions carefuly to ensure the correct proton version is being used and that the prefix is correctly setup before the installer attempts to modify it;

If nothing works, please post more system information. Specifically: linux distribution, Proton version, graphics card model, graphics driver version.

**Comment by Mass1milian0** [comment]
alright, after complitely purging mo2, vortex (i tried to use that to no avail), wine, winetrick and protontricks.

- verified game installation before continuing
- followed the installation from scratch, removed everything including the things in cache, only changing the installation path to match the drive the game is on
- launched the game and mo2 did **not** open at all, a black terminal appears in the top left of the screen for less than a second then just disappears, proton log do generate and is considerably shorter, here is it: https://pastebin.com/T0FhMCbU

- nuked mo2 installation again, starting over, repeated step 1 and 2 without changing installation path
- ran the game once again, same issue as before, very similar log: https://pastebin.com/VZeqVj2p

somehow i managed to make everything even worse than when i first started

here is my system information:
- linux distribution: Pop!_OS 21.10 64 bytes
- Proton version: 6.3-8
- graphics card model: Nvidia GeForce GTX 1060 dual 3gb (by asus)
- graphics driver version: 470.86

**Comment by rockerbacon** [comment]
>     * only changing the installation path to match the drive the game is on
> 
>     * launched the game and mo2 did **not** open at all, a black terminal appears in the top left of the screen for less than a second then just disappears, proton log do generate and is considerably shorter, here is it: https://pastebin.com/T0FhMCbU

Probably #285

**Comment by Mass1milian0** [comment]
i fixed that, it wasn't relatead at all to #285 the issue seems to be: installer forgot to put modorganizer.exe in the installation in the modorganizer installation folder, idfk anymore tbh, now mo2 launches however i had to reinstall skyrim before going into further testing as vortex left some leftover files in the data folder, i'll continues reporting once it finishes downloading and installing
